tests/data/dataset_readers/vqav2_test.py

```diff
@@ -53,3 +53,14 @@ def test_read(self):
         # Nothing should be masked out since the number of fake boxes is the same
         # for each item in the batch.
         assert tensors["box_mask"].all()
+
+    def test_read_without_images(self):
```

Comment:
What do we gain by allowing the ability to read without images?

Reply:
We use that to make vocab.

Reply:
Ah, ok. So the configuration of not calling `_process_image_paths` when reading serves this purpose.